cernopendata/modules/fixtures/data/docs/cern-open-data-release-2024/cern-open-data-release-2024.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "author": "CERN",
+    "body": {
+      "content": "cern-open-data-release-2024.md",
+      "format": "md"
+    },
+    "collections": [
+      {
+        "primary": "News"
+      }
+    ],
+    "date_published": "2024-01-31",
```

Comment:
Please add also `"featured": 1,` so that the news item will appear on the home page, and decreased `"featured"` values on other existing records.

See example: https://github.com/cernopendata/opendata.cern.ch/pull/3503/files